my-app/src/app/shared/pipes/duration-handler.pipe.spec.ts

```diff
@@ -0,0 +1,20 @@
+import {DurationHandlerPipe} from "./duration-handler.pipe";
+
+describe('DurationHandlerPipe', (): void => {
+  const pipe = new DurationHandlerPipe();
+  const DURATION_55MIN: number = 55
+  const DURATION_2H: number = 120;
+  const DURATION_135MIN: number = 135;
+
+  it(`should transform '${DURATION_55MIN}' to '${DURATION_55MIN} min'`, (): void => {
```

Comment:
In tests, it's usually ok to have some duplication. I would simply inline values you put in `const`s